xcodeproj/internal/pbxproj_partials.bzl

```diff
@@ -1092,6 +1094,9 @@ def _write_target_build_settings(
         swift_args,
         swift_debug_settings_to_merge = EMPTY_DEPSET,
         team_id = None,
+        xcode_config,
```

Comment:
@brentleyjones Got a question on handling this param in the unit test for this function. Right now, `apple_support.run()` would try to validate `xcode_config` and fails in unit test. 

I would like to mock `XcodeVersionConfig`, but can we even mock it?